libcodechecker/analyze/tidy_output_converter.py

```diff
@@ -81,7 +81,7 @@ class OutputParser(object):
         # Column number followed by a ':' and a space.
         r'(?P<column>\d+?):\ '
         # Severity followed by a ':'.
-        r'(?P<severity>\w+):'
+        r'(?P<severity>\b(?!note\b)\w+):'
```

Comment:
What it we would use a whitelist here `warning`, `error`?
Could clang-tidy produce other types of reports?